Makefile

```diff
@@ -12,8 +13,8 @@ PACKAGES=$(shell go list ./...)
 
 BUILD_TIME := $(shell date -u +%Y%m%d.%H%M%S)
 BUILD_HASH := $(shell git rev-parse HEAD)
-LDFLAGS += "-X github.com/mattermost/chimera.BuildHash=$(BUILD_HASH)"
-
+LDFLAGS += -X github.com/mattermost/chimera.BuildHash=$(BUILD_HASH)
+export GO111MODULE=on
```

Comment:
That should not longer be needed, should it?

Reply:
thanks for this comment. Agree.